firebase-firestore/src/main/java/com/google/firebase/firestore/local/Persistence.java

```diff
@@ -52,7 +52,7 @@
 
   /** Temporary setting for enabling document overlays. */
   // TODO(Overlay): Remove this.
-  public static boolean OVERLAY_SUPPORT_ENABLED = BuildConfig.ENABLE_OVERLAY;
+  public static boolean OVERLAY_SUPPORT_ENABLED = true;
```

Comment:
I understand I am just some random external person looking at PRs (why am I even here 😆 ?) but curious why not remove the entire symbol if the feature is shipping. Seems it should not be referenced anymore as there is no way to toggle it

Reply:
Ha, you are right this should be removed.

We are keeping it for now because there are more work associated with fully removing the old code, some old logic is no longer required and should be removed. But we want to keep the release PR small, that that work will be in a followup PR.

But thanks for the extra pair of eyes!